src/workerd/io/worker-entrypoint.c++

```diff
@@ -298,9 +300,16 @@ kj::Promise<void> WorkerEntrypoint::request(kj::HttpMethod method,
     TRACE_EVENT("workerd", "WorkerEntrypoint::request() run", PERFETTO_FLOW_FROM_POINTER(this));
     jsg::AsyncContextFrame::StorageScope traceScope = context.makeAsyncTraceScope(lock);
 
+    abortController = jsg::alloc<api::AbortController>();
+    kj::Maybe<jsg::Ref<api::AbortSignal>> signal;
+
+    KJ_IF_SOME(ctrl, abortController) {
+      signal = ctrl->getSignal();
+    }
```

Comment:
I think it's misleading to use `kj::Maybe` and `KJ_IF_SOME` in cases where the value can never actually be null -- it adds a lot of noise and cognitive overhead for the reader (and some redundant checks at runtime).

In this case you could avoid it by restructuring the code like this:

```suggestion
    jsg::Ref<api::AbortSignal> signal =
        abortController.emplace(jsg::alloc<api::AbortController>()).getSignal();
```